TDP/tdp-53.md

```diff
@@ -0,0 +1,110 @@
+---
+tdp: 53
+title: Funding Request - mStable Ecosystem subDAO (Jan-Apr 2023)
+status: Proposed
+author: Fungus (@fungalarmy)
+discussions-to: https://forum.mstable.org/t/funding-request-mstable-ecosystem-subdao-9-jan-apr-2023/966
+created: 2022-12-08
+---
+
+## Simple Summary
+
+This proposal requests four (4) months of operational funding for the mStable Ecosystem subDAO. The proposal will:
+
+⦁	Outline a budget proposal for the period of 1st January 2023 to 30th April 2023
+⦁	Define deliverables and methods of transparently reporting activity and performance during the funding period
+⦁	For more background on the mStable Ecosystem subDAO's purpose, mandate, structure, and internal processes; please see the initial funding request here [TDP 46](https://forum.mstable.org/t/tdp-46-mstabledao-restructure-ecosystem-subdao/895)

```

Comment:
Can we make these bullet points conform to Markdown?

https://github.com/tchapi/markdown-cheatsheet/blob/master/README.md

See bullet list item.